### ISSUE TITLE
How many epochs and what is the best loss of both models in the Chart-to-Table Translation stage?

### ISSUE THREAD

**Issue by tannq-jo** [OpenGVLab/ChartAst#15]
Hi,
I am trying to replicate the Chart-to-Table Translation stage as described in your paper. It's my first time pre-training such a big model on a large-scale dataset so I don't know when I should stop the pre-training loop. Could you please kindly provide the information about the number of epochs and the best loss that you pre-trained both ChartAst-D and ChartAst-S models in the Chart-to-Table Translation stage.

Thank you for your attention to this matter.

**Comment by FanqingM** [comment]
As far as I known, when training a large model, It needs Largerer batsh size, Lower learning rate, More Data And just one Epoch.
I always following this.

**Comment by tannq-jo** [comment]
Thank you for your information. May I ask you some additional questions:
1. How about the small model ChartAst-D? Did you train it for only one epoch?
2. What are the best losses of both models ChartAst-D and ChartAst-S? As our training data is different, the information about the best losses is really helpful for us to somehow determine the Upper Bound or the performance needed for better generalization, therefore, knowing when is the right time to stop the training loop.
